etc/kube/pachyderm.json

```diff
@@ -348,24 +346,7 @@
   },
   "status": {
     "replicas": 0
-  }
-}
-{
-  "kind": "Secret",
-  "apiVersion": "v1",
-  "metadata": {
-    "name": "amazon-secret",
-    "creationTimestamp": null,
-    "labels": {
-      "suite": "pachyderm",
-      "app": "amazon-secret"
-    }
   },
-  "data": {
-    "bucket": "",
-    "id": "",
-    "secret": "",
-    "token": "",
-    "region": ""
-  }
```

Comment:
This is a latent change from pachctl being updated without regenerating this file.